apispec/openapi-model/src/main/scala/sttp/tapir/openapi/OpenAPI.scala

```diff
@@ -164,28 +164,24 @@ case object ResponsesDefaultKey extends ResponsesKey
 case class ResponsesCodeKey(code: Int) extends ResponsesKey
 
 // todo: links
-case class Response(description: String, headers: ListMap[String, ReferenceOr[Header]], content: ListMap[String, MediaType]) {
+case class Response(description: String, headers: ListMap[String, ReferenceOr[Header]], content: ListMap[String, MediaType])
 
-  def merge(other: Response): Response =
-    Response(
-      description,
-      headers ++ other.headers,
-      content ++ other.content
-    )
+object Response {
+  val Empty: Response = Response("", ListMap.empty, ListMap.empty)
 }
 
 case class Example(summary: Option[String], description: Option[String], value: Option[ExampleValue], externalValue: Option[String])
 
 case class Header(
-    description: Option[String],
-    required: Option[Boolean],
-    deprecated: Option[Boolean],
-    allowEmptyValue: Option[Boolean],
-    style: Option[ParameterStyle.ParameterStyle],
-    explode: Option[Boolean],
-    allowReserved: Option[Boolean],
-    schema: Option[ReferenceOr[Schema]],
-    example: Option[ExampleValue],
-    examples: ListMap[String, ReferenceOr[Example]],
-    content: ListMap[String, MediaType]
+    description: Option[String] = None,
```

Comment:
do we have default arguments in other places as well? would be keep to stay consistent in the model

Reply:
no, adjusted